lib/cmps/onetrust.ts

```diff
@@ -10,6 +11,10 @@ export default class Onetrust extends AutoConsentCMPBase {
     super("Onetrust");
   }
 
+  runContext: RunContext = {
+    urlPattern: '^https://(www\\.)?nba\\.[^/]+//'
```

Comment:
this will cause the onetrust rule to trigger _only_ on nba.com. I don't think you meant this. Try using [negative lookaheads](https://stackoverflow.com/questions/1538512/how-can-i-invert-a-regular-expression-in-javascript)?

Reply:
@muodov thanks! Change made. Also FYI I changed the end of the regex and kept` .com` 
Looks like e.g. nba.fr is taken and an unrelated website.